scripts/asm-installer/install_asm

```diff
@@ -2192,14 +2192,21 @@ EOF
 
 is_user_cluster_admin() {
   local GCLOUD_USER; GCLOUD_USER="$(gcloud config get-value core/account)"
+  local IAM_USER; IAM_USER="$(iam_user)"
```

Comment:
You should use `local_iam_user` here, since we (almost) always want the identity of the client side and not any remote users.

Reply:
Done